client/components/ScenariosList/index.jsx

```diff
@@ -78,6 +81,7 @@ class ScenariosList extends Component {
                     <ScenarioEntries
                         scenarioData={this.state.scenarioData}
                         isLoggedIn={this.props.isLoggedIn}
+                        path={this.props.location.pathname.slice(1)}
```

Comment:
In this case, the value returned by `this.props.location.pathname.slice(1)` is no longer a "path", it's just the category, so I think `path` => `category`